src/components/text-input/index.md.njk

```diff
@@ -59,6 +59,19 @@ Fluid width inputs will resize with the viewport.
 
 {{ example({group: "components", item: "text-input", example: "input-fluid-width", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
+### Numbers
+
+If it’s likely that the user will need to enter a number and you want to bring up the numeric keypad on a mobile device, set the `inputmode` attribute to `numeric` and the `pattern` attribute to `[0-9]*`. See how to do this in the HTML and Nunjucks tabs in the following example.
```

Comment:
I don't think we point people at examples like this elsewhere ("See how to do this in the HTML and Nunjucks tabs in the following example") – is it required?

Reply:
@36degrees We do it in the [`autocomplete` example](https://design-system.service.gov.uk/components/text-input/#use-the-autocomplete-attribute) which is where we took the wording from.